app/assets/javascripts/discourse/app/components/discovery/filter-navigation.gjs

```diff
@@ -81,6 +124,462 @@ export default class DiscoveryFilterNavigation extends Component {
     this._allowEnterSubmit = !value;
   }
 
+  @action
+  setupEventListeners() {
```

Comment:
you can have the element here `setupEventListeners(element)` so you don't need the check